app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentRow.kt

```diff
@@ -139,7 +137,7 @@ data class OverdueAppointmentRow(
     holder.overdueDaysTextView.text = context.resources.getQuantityString(
         R.plurals.overdue_list_item_overdue_days,
         overdueDays,
-        overdueDays
+        "$overdueDays"
```

Comment:
This is un unnecessary template since it has nothing else apart from the variable. Can't this just be `overdueDays.toString()`?

Reply:
It can be. But I figured that this is as a consistent way to convert to strings. 
Any particular reason for suggesting this since both look the same to me.

Reply:
No specific reason, I personally use templates only when the variable is used for formatting. If you prefer this, that's fine as well.